### PR TITLE
[DOCS-3964] Add more details to troubleshooting section in Eventbridge guide

### PR DESCRIPTION
**Pull request by maycmlee**
## What does this PR do?

Add more detailed steps.

### Motivation

DOCS-3964

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 1]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
-    - If you are utilizing the events endpoint, you will need to include a `title` and `text` as `body.field` parameters in the API Destination connection. These are required values to `POST` to the events endpoint, see [Post an event API documentation][13]
-2. Once you've set up the destination, you can now follow the Amazon instructions to [create an EventBridge rule][9], where you set Datadog as your destination.
-3. Once you've set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration docs][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][10] in your account, use this AWS CloudShell command:
+    - If you are utilizing the events endpoint, you will need to include a `title` and `text` as `body.field` parameters in the API Destination connection. These are required values to `POST` to the events endpoint, see [Post an event API documentation][9]
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:21]:
```suggestion
    - If you are utilizing the events endpoint, you need to include a `title` and `text` as `body.field` parameters in the API Destination connection. These are required values to `POST` to the events endpoint. For more information, see the [Post an event documentation][9].
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 2]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
+2. Once you've set up the destination, you can now follow the Amazon instructions to [create an EventBridge rule][10], where you set Datadog as your destination.
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:22]:
```suggestion
2. Once you have set up the destination, see the Amazon documentation to [create an EventBridge rule][10], where you set Datadog as your destination.
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 3]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
+3. Once you've set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration docs][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][11] in your account, use this AWS CloudShell command:
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:23]:
```suggestion
3. Once you have set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration documentation][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][11] in your account, use this AWS CloudShell command:
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 4]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
 
     ```bash
     echo "test" > testfile.txt
     aws s3 cp testfile.txt s3://YOUR_BUCKET_NAME
     ```
-4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [logs console][11] or [events explorer][12], depending on which endpoint you are sending them to.
+4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [logs console][12] or [events explorer][13], depending on which endpoint you are sending them to.
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:29]:
```suggestion
4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [Logs Console][12] or [Events Explorer][13], depending on which endpoint you are sending them to.
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 5]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
 
 ## Troubleshooting
 
-To view the response of the API endpoints you can setup Amazon SQS for more details regarding the payloads sent to Datadog. 
-1. Create a new queue in Amazon SQS.
-2. In the **Target** section for your EventBridge rule, expand the **Additional settings** section. 
-3. In the **Dead-letter queue** section, choose **Select an Amazon SQS queue in the current AWS account to use as the dead-letter queue**.
-4. Select the AWS SQS created.
-5. Update the rule.
+You can set up an Amazon SQS queue to see more details regarding the payloads sent to Datadog and view the response of the API endpoints.
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:33]:
```suggestion
To see more details about the payloads sent to Datadog and to view the response of the API endpoints, set up an Amazon SQS queue:  
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 6]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
 ## Troubleshooting
 
-To view the response of the API endpoints you can setup Amazon SQS for more details regarding the payloads sent to Datadog. 
-1. Create a new queue in Amazon SQS.
-2. In the **Target** section for your EventBridge rule, expand the **Additional settings** section. 
-3. In the **Dead-letter queue** section, choose **Select an Amazon SQS queue in the current AWS account to use as the dead-letter queue**.
-4. Select the AWS SQS created.
-5. Update the rule.
+You can set up an Amazon SQS queue to see more details regarding the payloads sent to Datadog and view the response of the API endpoints. 
+1. Create a new queue in [Amazon SQS][14].
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:34]:
```suggestion
1. Create a queue in [Amazon SQS][14].
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 7]
```diff
@@ -18,24 +18,26 @@ Before you begin, you need a [Datadog account][2], with [an API key][3], and you
...
+2. Go to the [EventBridge rule][15] that you created in the [Configuration](#configuration) section.
+3. Select the **Targets** tab, and then click **Edit**.
```
Comment (alai97) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:36]:
```suggestion
3. Select the **Targets** tab and click **Edit**.
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some copy feedback!

**Comment by maycmlee** [comment]
Thanks @alai97! Accepted all suggestions, could you re-review?

**Review by alai97 — APPROVED** [review]

**Pull request merged**